test/server/utils.spec.js

```diff
@@ -222,18 +223,70 @@ describe( 'Utilities', () => {
     } );
 
     describe( 'toLocalMediaUrl function', () => {
-        const basePathDefault = config[ 'base path' ];
+        /** @type {import('sinon').SinonSandbox} */
+        let sandbox;
 
-        beforeEach(() => {
-            config[ 'base path' ] = 'http://enke.to';
-        });
+        beforeEach( () => {
+            sandbox = sinon.createSandbox();
```

Comment:
This change was more about test isolation (and consistency of same). There have been several cases where stateful setup/teardown like this has interfered with other tests.

I would **absolutely love** to make that change (and to generally pass arguments rather than depend on state like `config`). But I think I'd prefer to avoid major interface changes like that in such a narrowly scoped bugfix PR. With questionable test coverage and the potential for dynamic calls that could be hard to find, I think it's safer to hold off on that.